android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -273,15 +273,16 @@ public void refreshHiddenViews(boolean popup) {
                     toggleViewVisibility(curView, false, popup);
                 }
             } catch (Exception e) {
-                e.printStackTrace();
+                Timber.e(e);
             }
         }
     }
 
     @Override
     public void refreshSkipLogic(String parentKey, String childKey, boolean popup) {
         initComparisons();
-        for (View curView : skipLogicViews.values()) {
+        View[] views = skipLogicViews.values().toArray(new View[0]);
+        for (View curView : views) {
```

Comment:
This prevents a `ConcurrentModificationError` that occurs when one of the views is removed and the `foreach` loop calls next on the `Collection`